javav2/example_code/lookoutvision/src/main/java/com/example/lookoutvision/ListModelPackagingJobs.java

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+// snippet-start:[lookoutvision.java2.list_model_packaging_jobs.complete]
+
+package com.example.lookoutvision;
+
+import software.amazon.awssdk.services.lookoutvision.LookoutVisionClient;
+import software.amazon.awssdk.services.lookoutvision.model.LookoutVisionException;
+import software.amazon.awssdk.services.lookoutvision.model.ModelPackagingJobMetadata;
+
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+// Lists the model packaging jobs in an Amazon Lookout for Vision  project.
+public class ListModelPackagingJobs {
+
+    public static final Logger logger = Logger.getLogger(ListModelPackagingJobs.class.getName());
+
+    public static void main(String[] args) {
+
+        String projectName = null;
+
+        final String USAGE = "\n" + "Usage: " + "<project_name>\n\n"
+                + "Where:\n"
+                + "   project_arn - the project that contains the models packaging jobs that you want to list.\n\n";
```

Comment:
Capitalize leading "The" 

models --> model

Reply:
Fixed.